build_artifacts.py

```diff
@@ -118,6 +119,41 @@ def get_fields(self):
 
         return field_locations
 
+    def get_places(self):
+        """Get places data (e.g. Welcome Center, Challenge Course, etc.) via arcGIS API
+
+        :returns: Place locations
+        :rtype: dict
+        """
+        config = self.config['locations']['arcGIS']
+        url = f"{config['url']}{config['places']['endpoint']}"
+        params = config['fields']['params']
+        response = requests.get(url, params=params)
+
+        place_locations = []
+        ignored_places = []
+
+        if response.status_code == 200:
+            for feature in response.json()['features']:
+                attrs = feature['attributes']
+                # Only fetch the location if Prop_ID and uID are valid
+                if (
+                    utils.is_valid_field(attrs['Prop_ID'])
+                    and utils.is_valid_field(attrs['uID'])
+                ):
+                    place_location = PlaceLocation(feature)
+                    place_locations.append(place_location)
+                else:
+                    place_locations.append(attrs['OBJECTID'])
+
+            if ignored_places:
+                logger.warning((
+                    "These places OBJECTID's were ignored because they don't have"
+                    f"a valid Prop_ID or shouldn't be exposed: {ignored_places}\n"
```

Comment:
I believe you need a space either after `have` or before `a`

Reply:
Good catch, I also noticed there's a pep8 warning here. Fixed.